core/math/src/main/java/org/arakhne/afc/math/matrix/Matrix3d.java

```diff
@@ -2463,14 +2463,15 @@ private static int computeGr(double[] sValue, double[] eValue, double[] uValue,
             final double shift = computeShift(sValue[1], eValue[1], sValue[2]);
             double fvalue = (Math.abs(sValue[0]) - shift) * (dSign(cb48, sValue[0]) + shift / sValue[0]);
             double gvalue = eValue[0];
-            double rvalue;
 
-            rvalue = computeRot(fvalue, gvalue, sinr, cosr, 0);
```

Comment:
Don't remove this function since `computeRot` has a border effect.
You could remove the assignment to `rvalue` only.